owasp-top10-2017-apps/a1/copy-n-paste/README.md

```diff
@@ -1,40 +1,155 @@
 # CopyNPaste API
-> This is a simple Golang API that contains an example of an Injection vulnerability.
 
-<img src="images/CopyNPaste.png" align="center"/>
+<img src="../../../images/secDevLabs-logo.png" align="" height="80" />
```

Comment:
I think this first image should be about the app! Not necessarily the app's home page, but  maybe it's logo? It would be cool if we could add visual identity to each one of our apps, just like [Hack The Box][1] does with it's boxes. What do you guys think @vitoriario , @rafaeleyng ?

It doesn't need to be anything fancy, hack the box has a template and just changes some details to each box to kind of fit the theme. Here's an example:



![image](https://user-images.githubusercontent.com/40367872/52540013-2317f000-2d6c-11e9-874d-bfe9e405b466.png)

OR

![image](https://user-images.githubusercontent.com/40367872/52540019-31fea280-2d6c-11e9-90a3-6975e4f69cc7.png)

[1]:https://www.hackthebox.eu/

Reply:
@Krlier, we could use our logo. Something like:

![captura de tela 2019-02-11 as 10 28 45](https://user-images.githubusercontent.com/17754098/52563216-eab5f780-2de7-11e9-9571-e440f404589f.png)

What do you think?

Reply:
Using logos is a good idea, however it may take us some time to build "cool" logos for each app.    I may need some help doing so! 😃 

Also, we have to be careful when using "Difficult: Hard". How will we define what is `Hard` or what is `Easy`? Taking vulnhub.com apps as examples, I always had some trouble working on "Beginners" machines and went pretty well on "Medium/Hard" ones. The challenge may be easy for the author but not to whom does it.